src/main/resources/errors.yaml

```diff
@@ -833,6 +833,18 @@ request-errors:
       
       Resend the command using a supported value type.
 
+  - scope: SCHEMA
+    code: UNSUPPORTED_SCHEMA_NAME
+    title: The used schema name is not supported
+    body: |-
+      The command attempted to create a ${schemeType} with a name that is not supported.
+      
+      The supported ${schemeType} names must not be empty, more than ${nameLength} characters long, or contain non-alphanumeric-underscore characters.
```

Comment:
naming: `nameLength` -> `maxNameLength` ("nameLength" would sound like it refers to length of name being passed).